ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -9,7 +9,7 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 export AWS_SHARED_CREDENTIALS_FILE=$CLUSTER_PROFILE_DIR/.awscred
 export AZURE_AUTH_LOCATION=$CLUSTER_PROFILE_DIR/osServicePrincipal.json
 export GOOGLE_CLOUD_KEYFILE_JSON=$CLUSTER_PROFILE_DIR/gce.json
-export OS_CLIENT_CONFIG_FILE=${CLUSTER_PROFILE_DIR}/clouds.yaml
+export OS_CLIENT_CONFIG_FILE="${SHARED_DIR}/${OS_CLIENT_CONFIG_FILE}"
```

Comment:
`SHARED_DIR` is fine for secrets; it can be made read-only (for performance reasons) with `readonly_shared_dir: true` (see ipi-conf-openstack-precheck-ref.yaml)